java-extension/com.microsoft.java.test.plugin/src/main/java/com/microsoft/java/test/plugin/util/ProjectTestUtils.java

```diff
@@ -65,6 +65,11 @@
     }
 
     public static List<IClasspathEntry> getTestEntries(IJavaProject project) throws JavaModelException {
+        return getClasspathEntries(project, true);
+    }
+
+    public static List<IClasspathEntry> getClasspathEntries(IJavaProject project, boolean isTest)
```

Comment:
Just my 2 cents, I prefer making this a private function for better readability. And to be consistent with `getTestEntries`, using something like `getSourceEntries` as caller . E.g. 

```java
public static List<IClasspathEntry> getTestEntries(IJavaProject project) throws JavaModelException {
        return getClasspathEntries(project, true);
 }
public static List<IClasspathEntry> getSourceEntries(IJavaProject project) throws JavaModelException {
        return getClasspathEntries(project, false);
 }

private static List<IClasspathEntry> getClasspathEntries(IJavaProject project, boolean isTest)
```

Reply:
Makes sense